### ISSUE TITLE
Better error handling of out of bounds GET resources

### ISSUE THREAD

**Issue by aaron-suarez** [OperationCode/resources_api#69]
As of the latest commit, if you try to GET a resource with an id that's higher than the highest in the database, you get an error that looks like this:

```json
{
    "apiVersion": "1.0",
    "errors": [
        {
            "code": "something-went-wrong"
        }
    ],
    "status": "ok"
}
```

I put this error in here as a "catch all" for errors that we aren't specifically catching. We need to specifically catch the out-of-bounds scenario I described.

Acceptance criteria:
When doing `GET /api/v1/resources/10000000`, a 404 error should be returned.

**Comment by aaron-suarez** [comment]
Update: As of the latest commit, it now returns something like this:
```json
{
    "apiVersion": "1.0",
    "data": {},
    "status": "ok"
}
```

But it should still be a `404` not found instead

**Comment by aaron-suarez** [comment]
Fixed in #129
